src/js/pages/services/ServicesTab.js

```diff
@@ -108,6 +110,15 @@ var ServicesTab = React.createClass({
 
     // Find item in root tree and default to root tree if there is no match
     let item = DCOSStore.serviceTree.findItemById(id) || DCOSStore.serviceTree;
+    let groupId = ServicesPathUtils.getGroupFromAppId(item.id);
```

Comment:
We did (at least for the new structs), but had to revert that changes as it collides with the current `Item` struct and any refactoring of the item struct was out of scope. :(